tests/pykafka/test_simpleconsumer.py

```diff
@@ -44,7 +44,8 @@ def _get_simple_consumer(self, **kwargs):
         consumer.stop()
 
     def test_consume(self):
-        with self._get_simple_consumer() as consumer:
+        with self._get_simple_consumer(auto_offset_reset=OffsetType.EARLIEST,
+                                       reset_offset_on_start=True) as consumer:
```

Comment:
This test wasn't broken by the changes, but I wanted to make sure that if this one ever runs after `test_consumer_lag` that it doesn't fail. Since both of these tests consume the entire topic, I wanted to be sure this one starts from the beginning. Thinking about it though, they use different consumer groups, so I suppose this change shouldn't be necessary.